TreeDB/collections/column_physical_sortkey_pruning.go

```
@@ -0,0 +1,303 @@
+			return ranges, 0, true, columnSortKeyMarkFallbackNone, nil
+		}
+		ranges[i] = typedcolumn.Int64RangePredicate{Column: column.Definition.Name, Low: encoded, High: encoded}
+	}
+	return ranges, plan.PrefixLen, false, columnSortKeyMarkFallbackNone, nil
+}
+
+func encodeSortKeyPrefixPredicateValue(column typedColumnAdapterColumn, value string) (int64, bool, string, error) {
+	switch column.Field.ValueType {
+	case ColumnStoreValueString:
+		if err := validateTypedColumnAdapterStringDictionary(column, column.Definition.Cardinality, column.Dictionary); err != nil {
+			return 0, false, columnSortKeyMarkFallbackUncertifiedDictionaryOrdering, nil
```

golangci-lint (2.12.2)

[error] error is not nil (line 245) but it returns nil

(nilerr)